src/setup.py

```diff
@@ -44,18 +44,17 @@ def read(fname):
     ],
     install_requires=[
         'knack==0.1.1',
-        'msrest==0.4.26',
-        'msrestazure>=0.4.20',
+        'msrest==0.4.27',
+        'msrestazure==0.4.26',
         'requests',
         'azure-servicefabric==6.1.2.9',
```

Comment:
Do you not need to update this requirement as part of moving to 6.2?

Reply:
Yes, but since that wasn't out when I made the change, it would break this all together.

Can you take a look at this? https://github.com/Azure/azure-sdk-for-python/pull/2445 Is there anything we should be checking for, or if not, can you give it sign off? Do you have a list of things you check for?